index.html

```diff
@@ -5,19 +5,17 @@
 </head>
 
 <body>
-  <hello-world greet="Hoodie" if={showHoodie}></hello-world>
-  <hello-world greet="Titay"></hello-world>
-  <hello-world greet="Hyesoo"></hello-world>
-  <hello-world></hello-world>
-  <contact-list></contact-list>
+  <app></app>
 
-  <script src="bower_components/riot/riot.min.js"></script>
-  <script src="tags/hello-world.js"></script>
-  <script src="tags/contact.js"></script>
+  <script type="riot/tag" src="/tags/app.tag"></script>
+  <script type="riot/tag" src="/tags/contact-list.tag"></script>
+  <script type="riot/tag" src="/tags/hello-world.tag"></script>
+
+  <script src="bower_components/riot/riot+compiler.min.js"></script>
```

Comment:
when we load the `riot+compiler.min.js` instead of just the `riot.min.js`, then riot will compile the `*.tag` files on the fly. It makes it a little bit slower, but doesn’t matter to us :) It makes it so much simpler to work on a riot app :)